hello.py

```diff
@@ -0,0 +1,7 @@
+#!/Users/miaomiao/anaconda/bin/python
+
+def main():
+	print 'hello world'
+
+if __name__ == "__main__":
```

Comment:
you used single quote for 'hello world', but double quote for "__main__", inconsistent and messy xD